package.json

```diff
@@ -24,11 +24,11 @@
     "build": "run-s clean && run-s build:icons:all && run-s tsc && run-p build:rollup build:babel",
     "build:icons:all": "run-s clean && run-s build:icons build:icons:index",
     "build:icons": "run-p build:icons:emojis build:icons:glyphs build:icons:icons build:icons:logos build:icons:navicons",
-    "build:icons:emojis": "svgr --typescript src/svgs/emojis --out-dir src/icons/emojis/svgs",
-    "build:icons:glyphs": "svgr --typescript src/svgs/glyphs --out-dir src/icons/glyphs/svgs",
-    "build:icons:icons": "svgr --typescript src/svgs/icons --out-dir src/icons/icons/svgs",
-    "build:icons:logos": "svgr --typescript src/svgs/logos --out-dir src/icons/logos/svgs",
-    "build:icons:navicons": "svgr --typescript src/svgs/navIcons --out-dir src/icons/navIcons/svgs",
+    "build:icons:emojis": "svgr --typescript --svgo-config='{\"plugins\": {\"removeViewBox\": false}}' src/svgs/emojis --out-dir src/icons/emojis/svgs",
```

Comment:
`svgr`, courtesy of its underlying use of `svgo`, automatically removes `viewBox` if `height` and `width` are provided. We want to keep both, so that when a custom `height` and `width` is provided the image scales appropriately. 

https://react-svgr.com/docs/options/#svgo-config